data/mods/CrazyCataclysm/crazy_monsters.json

```diff
@@ -0,0 +1,74 @@
+[
+    {
+        "id": "mon_fungaloid",
+        "type": "MONSTER",
+        "name": "fungaloid",
+        "copy-from": "mon_fungaloid",
+        "special_attacks": [ [ "FUNGUS_CORPORATE", 30 ] ]
+    },
+    {
+        "id": "mon_zombie_electric",
+        "type": "MONSTER",
+        "name": "shocker zombie",
+        "description": "A human body with pale blue flesh, crackling with electrical energy. It seems eager to tell you something.",
+        "copy-from": "mon_zombie_electric",
+        "special_attacks": [ [ "SHOCKING_REVEAL", 25 ] ]
+    },
+    {
+        "id": "mon_zombie_skeltal",
+        "type": "MONSTER",
+        "name": "mr skeltal",
+        "description": "Devoid entirely of flesh and organs, this walking skeleton rattles you to the bone. In its bony hand, it holds a pristine trumpet, unmarred by the end of the world.",
+		"default_faction": "zombie",
```

Comment:
88 instances of "Tab" whitespace instead of four spaces. Starting here and ending line 72.